coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ServerResource.java

```diff
@@ -181,13 +190,18 @@ public Response<Map<String, Integer>> getNodeStatusTotal() {
     return execute(
         () -> {
           ClusterManager clusterManager = getClusterManager();
+          List<ServerNode> serverAllList = clusterManager.list();
           List<ServerNode> excludeNodes =
               clusterManager.getExcludeNodes().stream()
                   .map(exclude -> new ServerNode(exclude))
                   .collect(Collectors.toList());
+          List<ServerNode> activeServerList =
+              serverAllList.stream()
+                  .filter(node -> !excludeNodes.contains(node))
+                  .collect(Collectors.toList());
           Map<String, Integer> stringIntegerHash =
               Stream.of(
-                      clusterManager.list(),
+                      activeServerList,
                       clusterManager.getLostServerList(),
                       excludeNodes,
```

Comment:
Would it be the same if we just deleted this line?

Reply:
> Would it be the same if we just deleted this line?

The list() method gets all the server nodes, but the server nodes do not have a exclude state.